src/pages/LoginPage.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {NavLink, useHistory} from "react-router-dom";
+// Gebruikt bij eerdere opdracht
+// import Button from "../components/Button"; Gebruikt bij eerdere opdracht
+
+// Nu bij bonusopdracht nodig
+import {useForm} from "react-hook-form";
+import users from "../data/users.json"
+
+function LoginPage({inlog, toggleInlog}) {
+    const history = useHistory();
+    const {register, handleSubmit, formState: {errors}} = useForm();
+
+    let loginOK = false;
+    let loginWarning = false;
+
+    function toLogin() {
+        toggleInlog(true);
+        history.push("/");
+    }
+
+    // Nu bij bonusopdracht nodig
+    function checkInlog(data) {
+        for (let i = 0; i < users.length; i++) {
+            if ((data.user === users[i].username) && (data.pw === users[i].password)) {
+                loginOK = true;
+                break;
+            }
+        }
+        if (loginOK) {
+            toLogin();
+        }
+        else { loginWarning = true;
+        }
```

Comment:
Je zou dit in het vervolg wat kunnen versimpelen door een .some(x => ...) te gebruiken op de users array.